lcd/emonPiLCD.py

```diff
@@ -278,7 +279,7 @@ def updateLCD():
 
     elif page == 7:
         lcd[0] = "emonPi Build:"
-        lcd[1] = sd_image_version
+        lcd[1] = sd_image_version.decode('utf-8')
```

Comment:
encoding="utf-8" seems to be the default, at least on python 3.7. just `.decode()` is enough